src/metabase/models/permissions.clj

```diff
@@ -255,6 +258,15 @@
                                (and "schema/"
                                     (opt (and path-char "*/"
                                               (opt #"table/\d+/"))))))))
+               ;; any path starting with /general is a permission to access certain features of admin settings page
```

Comment:
nit: subscriptions are not a feature in the admin panel. maybe say something like `any path starting with /general is a permission that is not scoped by database or collection`? or something to that effect